include/soci/firebird/soci-firebird.h

```diff
@@ -84,6 +85,10 @@ struct firebird_standard_into_type_backend : details::standard_into_type_backend
 
     char *buf_;
     short indISCHolder_;
+
+private:
+    // Copy contents of a BLOB in buf_ into the given string.
+    void init_from_blob(std::string& out);
```

Comment:
IMO, `copy_from_blob` name would be clearer and would spare the comment

Reply:
OK, I'll change the function names to use `copy_` instead of `init_`. I'd rather keep the comment though, this code is not really obvious to follow.

Reply:
Sure